hw2/04-api.js

```diff
@@ -4,4 +4,18 @@ const url = 'https://restcountries.com/v3.1/all';
 
 // Add your code here
```

Comment:
I like how you set up the server with node. 
Overall, everything looked really well done.
Good job.